consumer.go

```diff
@@ -47,8 +47,15 @@ type ConsumerOptions struct {
 	BufferSize int
 	// Concurrency dictates how many goroutines to spawn to handle the messages.
 	Concurrency int
-	// RedisOptions is how you configure the underlying Redis connection. More
-	// info here: https://godoc.org/github.com/go-redis/redis#Options.
+	// RedisClient supersedes the RedisOptions field, and allows you to inject
+	// an already-made *redis.Client for use in the consumer.
+	RedisClient *redis.Client
+	// RedisOptions is deprecated, please use RedisClient instead.
```

Comment:
Wouldn't you need to import that package to create the `*redis.Options` to provide in the config?

Reply:
My questions aside, as requested I've removed the deprecation notice. 👍

Reply:
You don't need to be cause you can just use `redisqueue.RedisOptions` since [it's an alias](https://github.com/robinjoseph08/redisqueue/blob/13b26dc4b1d0a90b3fe4bfd239046fd7f0054dc6/redis.go#L15-L17) to `redis.Options`.